### PR TITLE
isl 0.24

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **isl/0.24**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/isl/all/conanfile.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
 class IslConan(ConanFile):
```
Comment (SpaceIm) [inline recipes/isl/all/conanfile.py:7]:
```suggestion

required_conan_version = ">=1.33.0"


class IslConan(ConanFile):
```

**Review by SpaceIm — CHANGES_REQUESTED** [review]

**Comment by SpaceIm** [comment]
Add this in autotools configuration:

```python
        if self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version) >= 12:
            self._autotools.flags.append("-FS")
```

**Review by SpaceIm — DISMISSED** [review]

**Comment by ericLemanissier** [comment]
`gmp.lib(bdiv_dbm1c.obj) : fatal error LNK1318: Unexpected PDB error; OK (0) 'C:\j\w\buildsinglereference@3\.conan\data\isl\0.22\_\_\build\048083ccb84e0809ad51b2a4017a45b658567c51\vc140.pdb'
`
`gmp.lib(tdiv_q_2exp.obj) : fatal error LNK1318: Unexpected PDB error; OK (0) 'C:\j\w\buildsinglereference@3\.conan\data\isl\0.22\_\_\build\048083ccb84e0809ad51b2a4017a45b658567c51\vc140.pdb'`

WAT ?

**Comment by ericLemanissier** [comment]
so same error as https://github.com/conan-io/conan-center-index/pull/7012. Does anyone have a clue ?

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]
LGTM

**Review by prince-chrismc — APPROVED** [review]

**Pull request merged**